### ISSUE TITLE
Create a Style Council intro page in SharePoint

### ISSUE THREAD

**Issue by lucyha** [nhsuk/nhsuk-service-manual#745]
## What

A new page in SharePoint to tell people about the Content Style Council. In the Content Community area of SharePoint, with links to Style Council meetings - and updates. Here: https://hscic365.sharepoint.com/sites/Content%20Community/SitePages/Home.aspx. 

## Why

The content community is developing a new area for the profession. That seems to be the place content designers want to find info about Style Council. Not in Confluence because Confluence isn't as widely accessible as SharePoint, but with a link from the service manual SharePoint site.

**Comment by chrimesdev** [comment]
@sarawilcox did we manage to do this?

**Comment by sarawilcox** [comment]
No, I don't think so @chrimesdev. Would be good to have some info in the Content Community section of SharePoint: https://hscic365.sharepoint.com/sites/Content
